src/vivarium_nih_us_cvd/constants/data_keys.py

```diff
@@ -378,6 +363,21 @@ def log_name(self):
 MEDIATION = __Mediation()
 
 
+class __JointPAFs(NamedTuple):
+    PAFS: TargetString = TargetString("risk_factor.joint.population_attributable_fraction")
```

Comment:
I never know what to call these oddball things...

Reply:
Maybe `"risk_factor.joint_mediated_risks.population_attributable_fraction"`?